lib/components/Alert/Alert.tsx

```diff
@@ -0,0 +1,221 @@
+  font-weight: ${({ theme }) => theme.alert.title.fontWeight};
+  line-height: ${({ theme }) => theme.alert.title.lineWeight};
+`;
+
+const Description = styled.span<Pick<AlertInlineProps, "status">>`
+  margin: 8px 0px;
+
+  font-size: ${({ theme }) => theme.alert.description.fontSize};
+  font-weight: ${({ theme }) => theme.alert.description.fontWeight};
+  line-height: ${({ theme }) => theme.alert.description.lineHeight};
+`;
+
+const Hint = styled.span`
+  display: flex;
+  align-items: center;
+  margin: 0px 10px;
+  cursor: pointer;
+
+  color: ${({ theme }) => theme.alert.hint.lineHeight};
+  font-size: ${({ theme }) => theme.alert.hint.fontSize};
+  font-weight: ${({ theme }) => theme.alert.hint.fontWeight};
+  line-height: ${({ theme }) => theme.alert.hint.lineHeight};
+  text-align: center;
+  text-decoration-line: underline;
+`;
+
+const AlertInline: FC<AlertInlineProps> = (props) => {
+  return (
+    <AlertWrapper status={props.status}>
+      <IconPanel status={props.status}>
+        <Icon path={mdiInformationOutline} size={0.7} />
+      </IconPanel>
+      <MainPanel>
+        <Title status={props.status}>{props.title}</Title>
+        <Description>{props.description}</Description>
+        <Actions>
+          {(props.status === "neutral" || !props.status) && (
+            <Button
+              onClick={() => {
+                props.onButtonClick && props.onButtonClick();
+              }}
+              buttonStyle="outlined"
+              variant="primary"
+            >
+              {props.labelButton}
+            </Button>
+          )}
+          {props.status === "primary" && (
+            <Button
+              onClick={() => {
+                props.onButtonClick && props.onButtonClick();
+              }}
+              buttonStyle="primary"
+              variant="primary"
+            >
+              {props.labelButton}
+            </Button>
+          )}
+          {props.status === "success" && (
+            <Button
+              onClick={() => {
+                props.onButtonClick && props.onButtonClick();
+              }}
+              buttonStyle="primary"
+              variant="primary"
+            >
+              {props.labelButton}
+            </Button>
+          )}
+          {props.status === "warning" && (
+            <Button
+              onClick={() => {
+                props.onButtonClick && props.onButtonClick();
+              }}
+              buttonStyle="secondary"
+              variant="primary"
+            >
+              {props.labelButton}
+            </Button>
+          )}
+          {props.status === "error" && (
+            <Button
+              onClick={() => {
+                props.onButtonClick && props.onButtonClick();
+              }}
+              buttonStyle="primary"
+              variant="destructive"
+            >
+              {props.labelButton}
+            </Button>
+          )}
+          <Hint
```

Comment:
@cld-santos this component is supposed to be a button, that would be better for accessibility and semantics.

and on figma it is an instance of the Button component, style tertiary, and variant according to the dialog (error, primary, etc).

You can add that with the existing Button component `<Button buttonStyle="tertiary" variant={variant} size="small">`

Reply:
I meant status, not variant, that's what you use.

Reply:
that's very good, because alert sticky has two buttons, so I'll hit two rabbits with only one "cajadada"